src/crimson/osd/object_context.h

```diff
@@ -271,6 +271,7 @@ class ObjectContextRegistry : public md_config_obs_t  {
 };
 
 std::optional<hobject_t> resolve_oid(const SnapSet &ss,
-                                     const hobject_t &oid);
+                                     const hobject_t &oid,
+				     bool check_clone_snaps);
```

Comment:
Let's change this to a default value of `check_clone_snaps = true`.
This should help us avoid some changes to `object_context_loader` and future users.

Reply:
Done.